x-pack/plugins/security_solution/server/endpoint/endpoint_app_context_services.ts

```diff
@@ -47,41 +40,13 @@ import {
 } from '../../common/experimental_features';
 
 export interface MetadataService {
-  queryStrategy(
-    savedObjectsClient: SavedObjectsClientContract,
-    version?: MetadataQueryStrategyVersions
-  ): Promise<MetadataQueryStrategy>;
+  queryStrategy(): MetadataQueryStrategy;
 }
 
-export const createMetadataService = (packageService: PackageService): MetadataService => {
+export const createMetadataService = (): MetadataService => {
```

Comment:
can we remove this and just import and call metadataQueryStrategyV2 directly where needed?

Reply:
sounds good to me. Use your gut on the refactor here, we have no attachments to the patterns this used